### PR TITLE
CLM-6003 Add documentation for AddResourceToQueue mutation

### PR DESCRIPTION
**Pull request by AlexONeillTI**

### CODE REVIEW

File: definition-map.js [hunk 1]
```diff
@@ -140,10 +140,37 @@ const QUERIES = {
             isPreview: "Requests the preview URL to launch SCORM content. This requires the current user to be in admin or manager role.",
             type: "The type of the Topic Or Course."
         }
+    },
+    CourseById: {
```
Comment (rjschill87) [inline definition-map.js:144]: Looks like some changes got carried over from your other PR -- mind stripping these out?

Comment (AlexONeillTI) [reply definition-map.js]: Removed and pushed

**Comment by AlexONeillTI** [comment]
@rjschill87 Regarding the `resourceId` argument, is it okay to provide the example of a possible `resourceId` the way I did, suggesting a `courseID`?

**Comment by rjschill87** [comment]
@AlexONeillTI Yeah, that's fine. Only change I'd suggest is `courseID` -> `Course ID` since `courseID` is more of an `arg` name

**Review by rjschill87 — APPROVED** [review]

**Pull request merged**